.gitignore

```
@@ -201,4 +201,5 @@ $RECYCLE.BIN/
 # End of https://www.toptal.com/developers/gitignore/api/rust,windows,macos,linux,visualstudiocode,jetbrains+all
 .env
 
-.buildconfig
\ No newline at end of file
+.buildconfig
+.env
```

```suggestion
.env
```
den er der allerede på linje 202